test_suite/cloud/test_azure.py

```diff
@@ -313,8 +313,17 @@ def test_rhui_certificate_date(self, host):
             cert_found = False
 
             if test_lib.is_rhel_sap(host):
-                cert_file = '/etc/pki/rhui/product/content-sap-ha.crt'
-                cert_found = host.file(cert_file).exists
+                possible_sap_cert_files = [
```

Comment:
Let's use the same variable to reduce code duplication.

```suggestion
                possible_cert_files = [
```